emails/DrawingEmail.tsx

```
@@ -0,0 +1,247 @@
+  Text,
+} from '@react-email/components';
+import { IGiftSuggestion } from '@/app/types/giftSuggestion';
+
+interface DrawingEmailProps {
+  userName: string;
+  recipient: string;
+  giftSuggestions: IGiftSuggestion[]
+}
+
+const handleAmazonLink = ({ searchTerm }: { searchTerm: string }) => {
+  const encodedSearch = encodeURIComponent(searchTerm).replace(/%20/g, '+');
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)